frontend/src/components/common/Header.tsx

```diff
@@ -8,51 +9,163 @@ import {
+            </Link>
+          </Text>
+          {authenticatedUser && (
+            <>
+              <Text>
+                <Link as={ReactLink} to={LANDING_PAGE}>
+                  {" "}
+                  {/* update this to account page */}
+                  My Account
+                </Link>
+              </Text>
+              <Button onClick={onLogOutClick} variant="link" color="black">
+                <Text>Log Out</Text>
+              </Button>
+            </>
+          )}
+          {!authenticatedUser && (
+            <Text>
+              <Link as={ReactLink} to={LOGIN_PAGE}>
+                Sign In
+              </Link>
+            </Text>
+          )}
+        </Stack>
       </Flex>
       <Drawer placement="left" onClose={onClose} isOpen={isOpen} size="xs">
         <DrawerOverlay />
         <DrawerContent backgroundColor="evergreen.100">
           <DrawerBody>
+            <IconButton
+              w="24px"
+              h="24px"
+              float="right"
+              mt="15px"
+              aria-label="close drawer"
+              backgroundColor="transparent"
+              onClick={onClose}
+            >
+              <CloseIcon color="white" />
+            </IconButton>
+            <Image
+              mt="70px"
+              mb="30px"
+              src="drawer-logo.png"
+              alt="Community Fridge logo"
+            />
             <Stack spacing="1rem">
               <Text color="squash.100" textStyle="mobileHeader4">
-                Schedule Dropoff
+                <Link as={ReactLink} to={LANDING_PAGE}>
+                  Home
+                </Link>
               </Text>
               <Text color="squash.100" textStyle="mobileHeader4">
-                <Link as={ReactLink} to={LOGIN_PAGE}>
-                  Sign In
+                <Link as={ReactLink} to={SCHEDULING_PAGE}>
+                  Schedule Donation
                 </Link>
               </Text>
+              {authenticatedUser && (
+                <Text color="squash.100" textStyle="mobileHeader4">
+                  <Link as={ReactLink} to={LANDING_PAGE}>
+                    {" "}
+                    {/* update this to account page */}
+                    My Account
+                  </Link>
+                </Text>
+              )}
+              {!authenticatedUser && (
+                <Text color="squash.100" textStyle="mobileHeader4">
+                  <Link as={ReactLink} to={LOGIN_PAGE}>
+                    Sign In
+                  </Link>
+                </Text>
+              )}
             </Stack>
+            {authenticatedUser && (
+              <Button
+                onClick={onLogOutClick}
+                variant="link"
+                position="fixed"
+                bottom="20px"
+              >
+                <Text color="squash.100" textStyle="mobileHeader4">
+                  Log Out
+                </Text>
+              </Button>
+            )}
```

Comment:
```suggestion
              {!authenticatedUser ? (
                  <Link as={ReactLink} to={LOGIN_PAGE}   
                   color="squash.100"
                  textStyle="mobileHeader4">
                    Sign In
                  </Link>
              )}
            </Stack>
            : (
              <Button
                onClick={onLogOutClick}
                variant="link"
                position="fixed"
                bottom="20px"
                color="squash.100" 
                textStyle="mobileHeader4"
              >
                  Log Out
              </Button>
            )}
```

Reply:
hmm i'm not sure if this works because of that closing stack tag in the middle there